src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -398,12 +409,16 @@ private void InitNativeProcess()
             // redirecting anything. This is a bit tricky as we always run redirected so
             // we have to see if the redirection is actually being done at the topmost level or not.
 
-            //Calculate if input and output are redirected.
-            bool redirectOutput;
-            bool redirectError;
-            bool redirectInput;
+            // Calculate if input and output are redirected.
+            bool redirectOutput = true;
+            bool redirectError = true;
+            bool redirectInput = true;
```

Comment:
Why do we want `redirectInput` to be true by default? Even if we are transcribing and screen scrape is not supported, we shouldn't need to always redirect the input.

Reply:
And form line 442 below, it indicates that we transcribe for native command only if `_runStandAlone` is true. Now it looks like you are assuming we can do transcribe regardless of `_runStandAlone`. That seems not right.